hushline/templates/base.html

```diff
@@ -140,6 +140,9 @@ <h1>🤫 Hush Line</h1>
                     <li><a href="{{ url_for('logout') }}">Logout</a></li>
                   </ul>
                 </div>
+                {% if is_premium_enabled and (user.tier_id == 1 or user.tier_id == None) %}
```

Comment:
`is none` (lowercase) is a valid test in jinja

Reply:
Fixed in d57e2adb3ce90717ed32aaf6fd96b5e1ec1c643b